### ISSUE TITLE
IA updated / designed Meetings view

### ISSUE THREAD

**Issue by ackernaut** [ParabolInc/parabol#4321]
### Background
We have data that the IA needs work. Folks need to find their meetings easier. Teams need to be easier to manage. The invitation flow can use improvement. We want to develop a full set of activities to support agile teams: new meeting types, async video updates, guided threads i.e. proposals/decisions, etc. A strong IA will become even more important as our system becomes more complex. It should be easier to interact with during more frequent sessions. It’ll stop being an IA used only to get to a sync meeting view every 2 weeks.

My hunch is to take a bottom-up approach to engage all users as the primary use case over team or org leads. They want to see where (activity, meeting) to give their attention, consume updates from the teammates, and have less need for an unstructured chat. We take learnings from engaging others on sites like Instagram, Tik Tok, Twitter. But we are not unstructured like Slack, Notion, Loom. Folks that form the teams should feel energized to interact with their team on our platform but in a purposeful, curated way (like Range). Team and org leads will use it because they have to.

### Acceptance Criteria
- Developed a clickable prototype that can be tested internally and externally that shows:
  - Primary use cases
    - How to clearly contribute to an open, active meeting or guided thread
    - How to see async video updates and respond in ways that map to other activities (e.g. add this to the next retro, roll this into a team update, etc.)
  - Secondary use cases
    - How to manage the team or an org (members, integrations, billing)

### Estimated Effort
- 20 hours

**Comment by ackernaut** [comment]
Closing #4019 in favor of this issue to continue work from S.P.A. 16

**Comment by jordanh** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/19482951) 🔐 shared [a video discussing where they'd like to see meeting history and how the timeline fails them today](https://drive.google.com/file/d/1CUZOESAu29y1ILE6Q0TUB0sfF-h13l-h/view?t=6m21s) 🔐

**Comment by jordanh** [comment]
@ackernaut could you include thoughts on a help section, including Live Chat in your exploration?

**Comment by ackernaut** [comment]
For review: [Loom](https://www.loom.com/share/4d2ec0cf9d4b4a76ba593907711e2e7c) + [Notion](https://www.notion.so/parabol/Meetings-view-c219517dc0d349e4ad57d0896c945dca#85cff74a507746988fb5f2b10b0de80f) + [Figma](https://www.figma.com/file/dfTQxhoKzohkCpxlztBbhS/Active-and-Scheduled-Meetings-Dashboard?node-id=37%3A2045)

### TL;DR
- We have a meetings view that shows ad hoc meetings and meeting series
- Completed meetings are still on the timeline
- TBD inline vs FAB control (or conditional e.g. FAB for mobile, inline for empty state, etc.)

### Next steps
- Drafted 2 implementation issues for a basic meetings view and a basic meeting card (including series stack)
- Created 3 design issues:
    1) designed empty state, first-time user state, just 1 meeting state
    2) designed an updated new (and edit) meeting view to include setting up and editing a series
    3) designed an IA concept that supports various activity types: meetings, updates, threads

**Comment by ackernaut** [comment]
Closed. We’re moving forward with designing first-time and empty states, and updates to how to create and edit meetings. We’ll capture work as implementing meeting cards, the meetings view, recurring meetings, and a first calendar integration.
